it/uk/gov/hmrc/transitmovementseisstub/connectors/EISConnectorSpec.scala

```diff
@@ -72,14 +74,16 @@ class EISConnectorSpec
     uriStub
   )
 
+  val clock: Clock = Clock.fixed(LocalDateTime.of(2023, 5, 4, 12, 36, 8, 0).toInstant(ZoneOffset.UTC), ZoneOffset.UTC)
```

Comment:
Its better to comments something about these number 2023, 5, 4, 12, 36, 8, 0. to make more sense to anyone looking on this.